ring/model.go

```diff
@@ -142,6 +151,28 @@ func (i *InstanceDesc) GetRegisteredAt() time.Time {
 	return time.Unix(i.RegisteredTimestamp, 0)
 }
 
+func (i *InstanceDesc) SetRegisteredAt(t time.Time) {
```

Comment:
Why is this change required? The registered timestamp should never change after an instance is added to the ring. Having an exported function to change the setting looks risky to me.

Reply:
I don't think this is needed anymore. Removing.